### DISCUSSION TITLE
Having uv venv work with jupyter workflows

### DISCUSSION THREAD

**Discussion by iphils** [giswqs/geospatial#66] [General]
If we seek to work using uv in jupyter notebook, we might want to `ipykernel `package as one of the dependencies for geospatial library? 
There is a discussion on this topic happening in uv repo [here](https://github.com/astral-sh/uv/pull/7625)

**Comment by giswqs** [comment]
ipykernel is already a dependency of geospatial. It is not listed on the page, but it will be installed when you install geospatial.
